cpp/test/Ice/objects/Test.ice

```diff
@@ -61,15 +61,15 @@ class D
     bool postUnmarshalInvoked;
 }
 
-["protected"] class E
```

Comment:
We should remove these tests since they were apparently all about testing protected.